templates/post-install-job-dbmigrate.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  # TODO: this should be done during pre phase but to do so need
```

Comment:
There is a word missing somewhere in that sentence.

Reply:
Clarified doc string.